Doc/reference/expressions.rst

```diff
@@ -1905,18 +1905,35 @@ Evaluation order
 
 .. index:: pair: evaluation; order
 
-Python evaluates expressions from left to right.  Notice that while evaluating
-an assignment, the right-hand side is evaluated before the left-hand side.
+Python evaluates expressions from left to right (except for conditional_expression).
+Notice that while evaluating an assignment, the right-hand side is evaluated before
+the left-hand side.
+
+Examples for left to right:
+
+   .. code-block:: python
+
+      2 + 3 * 4 / 2 - 1
+      7
+
+Following is an example of right to left conditional_expression evaluation example:
+
+   .. code-block:: python
+
+       'foo' if 1 else 'bar' if 0 else 'baz'
+       'foo'
 
 In the following lines, expressions will be evaluated in the arithmetic order of
-their suffixes::
+their suffixes:
+
+   .. code-block:: python
```

Comment:
You can just keep the `::`, it automatically creates a code block.

Reply:
A explicit code block has better emphasis compared to to the : 
Additionally too keep it consistent with the examples I wrote above 1927 I assumed its worth changing line 1927 too. 
If you think is not needed should I revise and push again?

Reply:
Only 18 files use [`.. code-block:: python`](https://github.com/search?q=repo%3Apython%2Fcpython+%22..+code-block%3A%3A+python%22&type=code), while 460 use [`::`](https://github.com/search?q=repo%3Apython%2Fcpython+%3A%3A+language%3AreStructuredText&type=code&l=reStructuredText). So, I think it's best to use `::`.

Reply:
Thank you for pointing out to the inconsistencies. Since, [.. code-block:: python](https://github.com/search?q=repo%3Apython%2Fcpython+%22..+code-block%3A%3A+python%22&type=code)
has better emphasis overall compared to [::](https://github.com/search?q=repo%3Apython%2Fcpython+%3A%3A+language%3AreStructuredText&type=code&l=reStructuredText)
I don't mind refactoring all the 460 instances of [::](https://github.com/search?q=repo%3Apython%2Fcpython+%3A%3A+language%3AreStructuredText&type=code&l=reStructuredText)
that said I would like to hear from a member of cpython team as well on the best course of action!

Reply:
... just use `::`. The indented block is clear enough.

Reply:
The original pr I made was for adding examples and clarity to the documentation.  I will wait for a member to review and approve or comment on that before I make any more changes. Thanks for the feedback

Reply:
Personally, I prefer `.. code-block:: python` when the snippets are long enough or if I don't want a colon to open my code-block.

There is a difference between `::` and `.. code-block:: python`. The first one uses the *default* highlighting and automatically adds a colon at the end of the sentence, namely this makes

```rst
Hello::

	...
```

equivalent to 

```rst
Hello:

.. code-block::

   ...
```

See https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks.